feature/api-impresiones (#2538)
mercadopago/devsite-docs: guides/redelcom/migrate-via-api.en.md

- [Create order](/developers/en/reference/order/in-person-payments/point/create/post): This endpoint allows you to create an order that contains payment transactions for Mercado Pago Point. You can associate it with the desired terminal using its ID.
- [Get order by ID](/developers/en/reference/order/in-person/point/get-order/get): This allows you to retrieve all the information about an order using the ID obtained in the response to its creation.
- [Cancel order by ID](/developers/en/reference/order/in-person-payments/point/cancel-order/post): This allows you to cancel an order created for Mercado Pago Point using the reference ID obtained in the response to its creation.

> - [Refund order](/developers/en/reference/order/in-person-payments/point/refund-order/post): This endpoint allows to create a refund for a payment transactions associated with an order for Mercado Pago Point.

Reviewer: ```suggestion
- [Refund order](/developers/en/reference/order/in-person-payments/point/refund-order/post): This endpoint allows to create a refund for payment transactions associated with an order for Mercado Pago Point.
```

## Print configuration

The Printing API provides a practical solution for connecting your systems and managing the printing of invoices and receipts, as well as custom prints, on the Point terminals you have configured. With this API, it is possible to guarantee a unified and efficient payment experience.

> This resource allows for the printing of receipts and electronic invoices (DTE) and custom prints directly from a point of sale (POS) through the API, using the integrated printer of Smart devices. This simplifies the payment process and quickly responds to the needs of your business.

Reviewer: ```suggestion
This resource allows printing receipts and electronic invoices (DTE) and custom prints directly from a point of sale (POS) through the API, using the integrated printer of Smart devices. This simplifies the payment process and quickly responds to the needs of your business.
```

### Printing of invoices and electronic receipts

> The following types of Electronic Tax Documents (DTE) in XML format are accepted:

Reviewer: ```suggestion
The following types of Documentos Tributarios Electrónicos (DTE) in XML format are accepted:
```

> WARNING
> 
> Important
>
> The sent DTE must be compatible with the formats defined by the [SII](https://www.sii.cl/servicios_online/3532-formato_xml-3811.html).

| Document Type                          | Description                                                                                     |
|----------------------------------------|-------------------------------------------------------------------------------------------------|

> | Affected Invoice (33) and Exempt (34) | Refers to the tax document that has legal validity before the Internal Revenue Service (SII). |

Reviewer: ```suggestion
| Affected Invoice (33) and Exempt (34) | Refers to the tax document that has legal validity before the Servicio de Impuestos Interno (SII). |
```

| Affected Receipt (39) and Exempt (41) | Refers to the document that the customer receives when making a purchase, having accounting and tax validity. |

### Integration configuration 

> To configure the integration of the prints, use the API to manage the queuing of each attempt, considering the specifications of each endpoint. 

Reviewer: ```suggestion
To configure the integration of the prints, use the API to manage the queuing of each intent, considering the specifications of each endpoint. 
```

Reviewer: Así es como mencionamos a los intentos actualmente en Point

> WARNING
>
> Important
>

> > The terminal must be configured in POS (Point of Sale) mode. 

Reviewer: Gabai, acá vale reiterar la Warning que tenemos más arriba sobre la terminal en POS mode y cómo hacerlo.

> > The terminal must be configured in POS (Point of Sale) mode. 

Reviewer: ```suggestion
> The terminal must be configured in PDV (Point of Sale) mode. 
```

> The available endpoints are:

Reviewer: Y acá considero que es necesario explicar qué hace cada endpoint, como hicimos en el subtítulo anterior

   - [Create Terminal Action](/developers/en/reference/order/in-person-payments/impressions/post)
   - [Get Action by ID](/developers/en/reference/order/in-person-payments/impressions/get)
   - [Cancel Action by ID](/developers/en/reference/order/in-person-payments/impressions_cancel/post) 
    

> Wait until the attempt reaches the terminal and the print is processed. If the print does not arrive automatically, press the "green button" to fetch the attempt manually.

Reviewer: ```suggestion
Wait until the printing intent reaches the terminal and the print is processed. If the print does not arrive automatically, press the "green button" to fetch the intent manually.
```

> ### Custom tags

Reviewer: Gabai, cómo ejecutan la impresión de custom tags usando la API? Con los mismos endpoints? Si fuera así, reordenaría la info.
